spalp.module

```diff
@@ -49,9 +49,13 @@ function spalp_entity_view(array &$build, EntityInterface $entity, EntityViewDis
     // Get the app ID.
     $app_id = $entity->get('field_spalp_app_id')->getString();
```

Comment:
Could there be a scenario where $app_id returns as empty? The function would continue running, would this cause an error further down the line?